docs/_panels/jenkins-jobs.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Jenkins Jobs
+description: Information about Jenkins Jobs
+author: Bitergia
+screenshot: sigils/jenkins-jobs.png
+created_at: 2020-03-11
+grimoirelab_version: 0.2.38
+layout: panel
+---
+
+This panel provides an overview of the jobs executed in the Jenkins instances.
+
+## Metrics
+
+From left to right and top to bottom, the metrics provided are:
+
+* **Summary**: total numbers of builds, jobs, nodes and build durations.
+* **Results**: a pie chart that summarizes the proportions of the build results (success, failure, unstable, aborted).
+* **Jenkins projects**: a pie chart that highlights the builds by project.
```

Comment:
I'd rename it to `Projects and Jobs` and would count jobs instead of builds to align results with the dashboard's main use case.